components/places/src/places.udl

```diff
@@ -76,7 +76,7 @@ interface PlacesConnection {
     sequence<HistoryHighlight> get_history_highlights(HistoryHighlightWeights weights, i32 limit);
 
     [Throws=PlacesApiError]
-    void note_history_metadata_observation(HistoryMetadataObservation data);
+    void note_history_metadata_observation(HistoryMetadataObservation data, NoteHistoryMetadataObservationOptions options);
```

Comment:
I tried to put the default argument value here, but:

1. `optional NoteHistoryMetadataObservationOptions options` didn't work—it generated source code identical to `NoteHistoryMetadataObservationOptions options` for both Swift and Kotlin.
2. `optional NoteHistoryMetadataObservationOptions options = {}` raised a "No support for literal" error.

So they're in the Kotlin and Swift wrappers instead!